src/db_options.rs

```diff
@@ -1872,17 +2370,34 @@ impl Default for WriteOptions {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum ReadTier {
+    /// Reads data in memtable, block cache, OS cache or storage.
+    All = 0 as isize,
+    /// Reads data in memtable or block cache.
+    BlockCache = 1 as isize,
```

Comment:
Are these `as isize` casts required? As far as I can see, these values are transformed into `c_int` that isn't the same as `isize`. I guess `i32` is more appropriate here, but perhaps these casts can be simply omitted?